central/license/datastore/datastore_impl_test.go

```diff
@@ -81,10 +86,13 @@ func (s *licenseDataStoreTestSuite) TestEnforcesUpsert() {
 }
 
 func (s *licenseDataStoreTestSuite) TestAllowsUpsert() {
-	s.storage.EXPECT().UpsertLicenseKeys(gomock.Any()).Return(nil)
+	s.storage.EXPECT().UpsertLicenseKeys(gomock.Any()).Return(nil).Times(2)
 
 	err := s.dataStore.UpsertLicenseKeys(s.hasWriteCtx, []*storage.StoredLicenseKey{})
 	s.NoError(err, "expected no error trying to write with permissions")
+
+	err = s.dataStore.UpsertLicenseKeys(s.hasWriteCtx, []*storage.StoredLicenseKey{})
```

Comment:
`s.hasWriteAccessCtx` instead of `s.hasWriteCtx`
Also applies to `TestAllowsDelete`

Should there be a change in `TestAllowsList` as well ?